### PR TITLE
Add formatter

### PR DESCRIPTION
**Pull request by manumerous**
Adds scripts and a github actions check to ensure that the code is always formatted according to the black formatter.

### CODE REVIEW

File: .github/workflows/check_style.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Install requirements
+      run: pip install -r requirements.txt
+    - name: Check Code format
+      working-directory: Tools
```
Comment (Jaeyoung-Lim) [inline .github/workflows/check_style.yaml]: This directory doesn't exist ?

**Review by Jaeyoung-Lim — CHANGES_REQUESTED** [review]

File: .github/workflows/check_style.yaml (hunk 1 above)
Comment (Jaeyoung-Lim) [reply .github/workflows/check_style.yaml]: Ah, wait you added it in the PR :joy:

File: .github/workflows/check_style.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+      run: |
+        ls -la
+        chmod +x ./check_code_style.sh
```
Comment (Jaeyoung-Lim) [inline .github/workflows/check_style.yaml]: git tracks executable permissions, so you can just commit it

**Review by Jaeyoung-Lim — CHANGES_REQUESTED** [review]

**Comment by manumerous** [comment]
Your so fast Jay.... I was still working on it. Should have used a draft.

File: .github/workflows/check_style.yaml (hunk 1 above)
Comment (manumerous) [reply .github/workflows/check_style.yaml]: yeah there was a typo i overlooked in the naming of the script....

File: .github/workflows/check_style.yaml (hunk 2 above)
Comment (manumerous) [reply .github/workflows/check_style.yaml]: Thanks. Done!

**Review by Jaeyoung-Lim — APPROVED** [review]
Awesome!

**Pull request merged**